src/components/button/Button.scss

```diff
@@ -18,7 +18,7 @@
         background-color: var(--playkit-primary-darker-color);
       }
       &:focus {
-        background-color: var(--playkit-primary-brighter-color);
+        background-color: var(--playkit-primary-darker-color);
```

Comment:
do we want to change this for ALL button components ?
are product aware of that ?

Reply:
Yes, we already have 2 tickets solved by this fix. Color contrast requirements are not met otherwise. I've informed Sagi Bettan about this change